conf/gulp-tasks/templates.gulpfile.js

```diff
@@ -100,5 +126,11 @@ function watchTemplates (cb) {
   }, series(precompileTemplates, bundleTemplates));
 }
```

Comment:
Should we update the watchTemplates task as well?

Reply:
Very good call, updating